### ISSUE TITLE
Your script.module.inputstreamhelper fails with ism

### ISSUE THREAD

**Issue by Paco8** [matthuisman/slyguy.addons#489]
At least on kodi 19 (I didn't test with other versions) your modified version of script.module.inputstreamhelper returns false when the protocol is ism, so playback won't work:

```
  import inputstreamhelper
  is_helper = inputstreamhelper.Helper('ism', drm='com.widevine.alpha')
  if not is_helper.check_inputstream():
    xbmcgui.Dialog().notification('Error', 'inputstream not installed', xbmcgui.NOTIFICATION_ERROR, 5000)
    return

```

It can be tested with the attached sample addon.
[plugin.video.ism-test-0.0.4.zip](https://github.com/matthuisman/slyguy.addons/files/11937282/plugin.video.ism-test-0.0.4.zip)

**Comment by matthuisman** [comment]
what platform is that on?
ISM is Android only in Kodi right?

require_version(self.minversion, required=True) and KODI_VERSION > 17 and xbmc.getCondVisibility('system.platform.android')

**Comment by Paco8** [comment]
Ism is a manifest format. I tested it on Linux. The normal inputstreamhelper works fine with ism.

**Comment by matthuisman** [comment]
oh wicked. the last I knew it was android only :)

**Comment by matthuisman** [comment]
fixed in [v0.67.6](https://github.com/matthuisman/slyguy.addons/commit/fc50121ae2c61e4362f3cd5d462d9b0238bb7c9e)

Thanks for letting me know and providing the  test addon
